doc/book/sql-ddl.md

```diff
@@ -87,8 +87,21 @@ $table->changeColumn('name', Column\Varchar('new_name', 50));
 You may also *drop* existing columns or constraints:
 
 ```php
+use \Zend\Db\Metadata\Object\ConstraintObject;
+
 $table->dropColumn('foo');
-$table->dropConstraint('my_index');
+
+$constraint = new ConstraintObject('my_index', null);
+$table->dropConstraint($constraint);
+```
+
+Notice: On MySQL, you need to specify the type of constraint you want to drop:
+```php
+use \Zend\Db\Metadata\Object\ConstraintObject;
+
+$fkConstraint = new ConstraintObject('my_fk', null);
+$fkConstraint->setType('FOREIGN KEY');
```

Comment:
Oh I just noticed how ConstraintObject lets you do that... Why such duality... :( I guess at the moment both would work, but to go forward initiative to migrate to PHP 7.1 and all related type inconsistency fixes as discussed on development channel, this lose handling needs to go away. Imagine others who will not notice this duplication of functionality in ConstraintObject and ForeignKey.